### ISSUE TITLE
Invisible Keyboard Modifier Input

### ISSUE THREAD

**Issue by travankor** [labwc/labwc#2421]
### Problem description

Occasionally labwc enters a state where it adds a "ctrl" modifier to every key I press, making keyboard input useless. Switching to a tty and then back to the compositor seems to reset this state.

### Steps to reproduce

I don't know a good way to reproduce. Might have to do with having multiple keyboard layouts.

### labwc build source

Release

### labwc version

0.8.1

### labwc environment

From a TTY or some display manager like lightdm

**Comment by Bluey26** [comment]
Hi, the other day i experienced something like this. In my case, there was something wrong, i was unable to type my password correctly, and the issue was also present in my display manager (SDDM). I had to reboot to solve it.

It may not be the same thing (i would expect SDDM to not be affected by labwc) but i just comment it because it happened to me 2 times, randomly.

**Comment by travankor** [comment]
>Do you mean switching tty with key combos like Alt-Ctrl-F2?

Exactly. However, in this case I only need Alt-F2 because Ctrl is being automatically added. Convenient that the bug does not break this easy workaround.

>Could you show me your environment variables (e.g. ~/.profile or ~/.config/labwc/environment). Also, showing ~/.config/labwc/rc.xml might be helpful since the issue might have something to do with keybinds or mousebinds.

I have not done much customizing since I'm still testing out labwc.

`~/.config/labwc/environment`:
```
 XKB_DEFAULT_LAYOUT=us,workman
 XKB_DEFAULT_OPTIONS=grp:alt_shift_toggle
 XKB_MODEL=chromebook
```

**Comment by fungilife** [comment]
@travankor are you also using a DM?

I have never encountered such a problem, the only issue I have ever had was my fault of assigning a keybind to rc.xml that matched a shortcut in my terminal.  Having the behavior reset itself when going from tty to labwc may have to do with a bug in DM and not labwc.  Have you tried starting labwc as user from console disabling the DM?

**Comment by travankor** [comment]
there's no DM at all. just seatd + launch labwc on a console

**Comment by tokyo4j** [comment]
Sorry, I have no clue what the cause is. #2455 _may_ fix the issue, but being able to switch VTs without pressing Ctrl sounds really strange and I think there's something else broken.

Please notify us if you find a way to reproduce the issue.

**Comment by orfeasxyz** [comment]
I had the exact same problem on my laptop and pulling the control key and placing it back fixed it for good. It sounds unlikely for this to be a compositor issue.
